src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrInfo.cs

```diff
@@ -90,6 +95,8 @@ public VmrInfo(string vmrPath, string tmpPath) : this(new NativePath(vmrPath), n
 
     public LocalPath GetRepoSourcesPath(SourceMapping mapping) => VmrPath / SourcesDir / mapping.Name;
```

Comment:
Let's not duplicate then
```suggestion
    public LocalPath GetRepoSourcesPath(SourceMapping mapping) => GetRepoSourcesPath(mapping.Name);
```